colorbleed/plugins/maya/create/colorbleed_model.py

```diff
@@ -8,3 +10,14 @@ class CreateModel(avalon.maya.Creator):
     label = "Model"
     family = "colorbleed.model"
     icon = "cube"
+
+    def __init__(self, *args, **kwargs):
+        super(CreateModel, self).__init__(*args, **kwargs)
+
+        # create an ordered dict with the existing data first
+        data = OrderedDict(**self.data)
```

Comment:
@aardschok I think with [this change](https://github.com/getavalon/core/commit/3ecadff13427ff8239bca73168676902d028097c) in core, merged with [this PR](https://github.com/getavalon/core/pull/326) we wouldn't need to put this hack in code there, since it is an OrderedDict by default. :)

Haven't tested it though.

Reply:
Completely forgot about this :) thanks for pointing towards it. I will update some creator plugins :)